Articles of Governance/Document Standardization.md

```diff
@@ -33,7 +33,7 @@ For documents that were not versioned prior to the passing of this act the Grand
 ## Article 2 - Maintenance Standardization
 
 ### Section I
-For the sake of preservation of April Knight history, the Grandmaster will appoint one Upper Council member, other than the Grand Inquisitor, with the charge of maintaining April Knight documents to be known as the Scribe. This is not considered to be a seat on the Upper Council.
+For the sake of preservation of April Knights history, the First Herald will appoint any officer ranked Captain or higher, other than the Grand Inquisitor, with the charge of maintaining April Knights documents. This Knight will be known as the Scribe. They may choose themselves if it stands without objection in the Upper Council. The Scribe position is not an Upper Council position.
```

Comment:
Is this intended to be a simple majority or unanimous vote for the position of scribe in the UC?